Customer.io project/components/Sidebar.tsx

```
@@ -73,23 +73,31 @@ export const Sidebar: React.FC<SidebarProps> = ({
     sectionKey: string,
     icon: React.ReactNode
   ) => (
-    <div className="mb-4">
+    <div className="mb-4 dark:bg-[#25293c]">
       <button
         onClick={() => toggleSection(sectionKey)}
-        className="flex items-center justify-between w-full p-2 text-left hover:bg-gray-50 rounded-lg"
+        className="flex items-center justify-between w-full p-2 text-left hover:bg-gray-50 rounded-lg group"
       >
         <div className="space-x-2 flex items-center justify-start">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)